roles/callhome_configure/tasks/main.yml

```diff
@@ -1,5 +1,5 @@
 ---
 # tasks file for cluster
 - import_tasks: configure.yml
-  when: scale_callhome_params is defined and scale_callhome_params.is_enabled|bool
+  when: scale_callhome_params is defined and (scale_callhome_params.is_enabled|bool or (not scale_callhome_params.is_enabled|bool))
```

Comment:
don't make any change in the existing , add new task with new file with new flag as suggested in the previous comment,